Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.h

```diff
@@ -132,6 +132,8 @@ class WEBCORE_EXPORT ScrollingTreeScrollingNode : public ScrollingTreeNode {
     OverscrollBehavior horizontalOverscrollBehavior() const { return m_scrollableAreaParameters.horizontalOverscrollBehavior; }
     OverscrollBehavior verticalOverscrollBehavior() const { return m_scrollableAreaParameters.verticalOverscrollBehavior; }
 
+    NativeScrollbarVisibility horizontalScrollbarVisibility() const { return m_scrollableAreaParameters.horizontalNativeScrollbarVisibility; }
+    NativeScrollbarVisibility verticalScrollbarVisibility() const { return m_scrollableAreaParameters.verticalNativeScrollbarVisibility; }
```

Comment:
Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.h:136: error: [build/webcore_export] Inline functions should not be in classes annotated with WEBCORE_EXPORT. Remove the macro from the class and apply it to each appropriate method, or move the inline function definition out-of-line.
Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.h:137: error: [build/webcore_export] Inline functions should not be in classes annotated with WEBCORE_EXPORT. Remove the macro from the class and apply it to each appropriate method, or move the inline function definition out-of-line.